docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,240 @@
+          "type": "Long",
+          "length": "5"
+        },
+        {
+          "name": "active",
+          "type": "Boolean",
+          "length": "5",
+          "trueValue": "true",
+          "falseValue": "false"
+        },
+        {
+          "name": "optimized",
+          "type": "Boolean",
+          "length": "1",
+          "trueValue": "y",
+          "falseValue": "n"
+        },
+        {
+          "name": "parentCampaign",
+          "type": "String",
+          "length": "10"
+        },
+        {
+          "name": "weatherTargeted",
+          "type": "Character",
+          "length": "1"
+        }
+      ]
+    }
+    </value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application.
+As explained earlier padding is a character used to specify padding used in the incoming records to fill the alloted maximum space if required. User has the flexibility to specify a single padding character for the entire file or the user can choose to provide separate padding character for separate fields (columns of the record). Similar to padding character user also has the flexibility to define the alignment of the incoming records, user can choose whether the record is left,centre or right aligned.
+
+The sample json schema for a records having 3 fields as 'Occupation', 'Age' and 'Gender' having corresponding maximum space as 20,2 and 6, with corresponding padding character as '#','$' and '@' and with corresponding alignment as 'left', 'centre' and 'right' will be as follows:
+```
+{
+  "fields":
+  [
+    {
+      "name": "Occupation",
+      "type": "String",
+      "length": "20",
+      "padding": "#",
+      "alignment":"left"
+    },
+    {
+      "name": "Age",
+      "type": "Integer",
+      "length": "2",
+      "padding": "$",
+      "alignment":"centre"
+    },
+    {
+      "name": "Gender",
+      "type": "String",
+      "length": "6",
+      "padding": "@",
+      "alignment":"right"
+    }
+}
+```
+
+The corresponding record with values for 'Occupation' as Engineer, 'Age' as 30 and 'Gender' as Male would be as follows:
+```
+Engineer############30@@Male
+```
+
+
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+
+User can set this property in properties.xml in the following way
+
+```xml
+<property>
+    <name>dt.application.{ApplicationName}.operator.{OperatorName}.port.out.attr.TUPLE_CLASS</name>
+    <value>com.datatorrent.tutorial.fixedwidthparser.Ad</value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application and "com.datatorrent.tutorial.fixedwidthparser.Ad" is the fully qualified name of the Tuple class.
```

Comment:
This sentence can be move above the XML example and modified slightly:

"In the examples below, {OperatorName} is the name of the Operator, {ApplicationName} is the name of the application and "com.datatorrent.tutorial.fixedwidthparser.Ad" is the fully qualified name of the Tuple class."

That way, the duplicated description below can be omitted.